sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-part-time-process-integration.scheduler.models.ts

```diff
@@ -0,0 +1,84 @@
+import { OfferingIntensity, RelationshipStatus } from "@sims/sims-db";
+
+/**
+ * Values part of an MSFAA file record.
+ */
+export interface MSFAATestInputData {
+  msfaaNumber: string;
+  sin: string;
+  institutionCode: string;
+  birthDate: string;
+  surname: string;
+  givenName: string;
+  gender: string;
+  maritalStatus: RelationshipStatus;
+  addressLine1: string;
+  addressLine2: string;
+  city: string;
+  provinceState: string;
+  postalCode: string;
+  country: string;
+  phone: string;
+  email: string;
+  offeringIntensity: OfferingIntensity;
+}
+
+export const MSFAA_PART_TIME_MARRIED: MSFAATestInputData = {
+  msfaaNumber: "1000",
+  sin: "900000000",
+  institutionCode: "ABCD",
+  birthDate: "1995-06-30",
+  surname: "Doe",
+  givenName: "John",
+  gender: "male",
+  maritalStatus: RelationshipStatus.Married,
+  addressLine1: "Address Line 1",
+  addressLine2: "Address Line 2",
+  city: "Calgary",
+  provinceState: "AB",
+  postalCode: "H1H 1H1",
+  country: "Canada",
+  phone: "1111111111",
+  email: "john.doe@somedomain.com",
+  offeringIntensity: OfferingIntensity.partTime,
+};
+
+export const MSFAA_PART_TIME_OTHER_COUNTRY: MSFAATestInputData = {
+  msfaaNumber: "1001",
+  sin: "900000001",
+  institutionCode: "EFDG",
+  birthDate: "2000-01-01",
+  surname: "Other Doe",
+  givenName: "Jane",
+  gender: "unknown",
+  maritalStatus: RelationshipStatus.Single,
+  addressLine1: "Address Line 1",
+  addressLine2: undefined,
+  city: "Some city on United States",
+  provinceState: "BC",
```

Comment:
mm, these are added by the students, right? I believe we don't have `provinceState` when the country is different right?

Reply:
Set `provinceState` as `undefined`.